programs/quickstart.eve

```diff
@@ -0,0 +1,16 @@
+This will be where you write your web app for the quickstart!
```

Comment:
I would just add a header, since some people at the meetup were editing in markdown editors.